inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -25,16 +26,16 @@ class SourceParameters:
 
 @dataclass(kw_only=True, frozen=True)
 class ApplicationAddSourceParameters:
```

Comment:
Ensure that the constants and types used throughout the source management commands are consistent with the new optional GPG key parameters. [medium]